configs/components/oifs/oifs.yaml

```diff
@@ -193,6 +209,7 @@ nlev: ${levels_number}
 prepifs_expid: gu5a
 prepifs_startdate: 19820101
 mip: "cmip5"
+o3_scheme: "cmip6"
```

Comment:
Wouldn't it be better that this is set to `null` and to specify the scenario from the yaml of FOCI? In this way we don't interfere with the AWICM3 configuration, which is mostly based on getting the namelist with most of the predefined parameters.

Reply:
`o3_scheme: "cmip6"` is the default. Joakim added the new option `cariolle`. So setting `"cmip6"` here is required to be backward compatible.

Reply:
This option adds the changes to the tested `fort.4` in AWICM3, for example https://github.com/esm-tools/esm_tools/actions/runs/3134531104/jobs/5089180243#step:8:1023

I think we need the input from @JanStreffing, in any case, if this is a problem in the end, it's as simple as defining `o3_scheme: null` and setting it `o3_scheme: cmip6` as default in `foci-oifs.yaml`